runtime/polkadot/src/lib.rs

```diff
@@ -1626,7 +1627,8 @@ impl OnRuntimeUpgrade for FixCouncilDepositMigration {
 
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<(), &'static str> {
-		Self::execute(true)
+		Self::execute(true);
+		Ok(())
```

Comment:
The changes here and the parallel code in the kusama runtime should be superseded by whatever gets merged by https://github.com/paritytech/polkadot/pull/4692